test_fms/mpp/test_read_input_nml2.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+
+#***********************************************************************
+#                   GNU Lesser General Public License
+#
+# This file is part of the GFDL Flexible Modeling System (FMS).
+#
+# FMS is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or (at
+# your option) any later version.
+#
+# FMS is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with FMS.  If not, see <http://www.gnu.org/licenses/>.
+#***********************************************************************
+
+# This is part of the GFDL FMS package. This is a shell script to
+# execute tests in the test_fms/mpp directory.
+
+# Colin Gladue 04/27/2020
+
+touch test_numb_base.nml
+echo "&test_read_input_nml_nml" > test_numb_base.nml
+echo "test_numb = 0" >> test_numb_base.nml
+echo "/" >> test_numb_base.nml
+
+# Test 1
+sed "s/test_numb = [0-9]/test_numb = 1/" test_numb_base.nml > test_numb.nml
+cp $top_srcdir/test_fms/mpp/input_base.nml input.nml
+./test_read_input_nml.sh
```

Comment:
try using the full path here as well 

`./$top_srcdir/test_fms/mpp/test_read_input_nml.sh`